src/test/shell/integration/java_integration_test.sh

```diff
@@ -270,7 +270,7 @@ EOF
   cp ${PRODUCT_NAME}-bin/$pkg/java/hello/hello_deploy.jar $pkg/ugly/
 
   $pkg/ugly/hello build.target build.time build.timestamp \
-      main.class=hello.Hello "$expected_build_data" 2>&1 >>$TEST_log
+      main.class=hello.Hello "$expected_build_data" >> $TEST_log 2>&1
```

Comment:
This might be critical, can someone tell me, what this is supposed to do?
A) "Point stderr to where stdout is currently pointing (the terminal). Then point stdout to /dev/null".
B) "Point stdout to /dev/null. Then point stderr to where stdout is currently pointing (/dev/null)".
See [SC2069](https://github.com/koalaman/shellcheck/wiki/SC2069) for reference.

Reply:
I don't think that matters, why would it? This is just trying to get everything appended to TEST_log, not sure why it is appending rather than overwriting thought.

Reply:
Quick question: Are there tests being run over the contents of this file?
Because then it would be important to know which stream goes where.

Reply:
The expect_log behind looks for 'Hello, World!' in that output, apart from that this file is shown by the test framework if the test fails.

Reply:
So, would you like to redirect `stderr` to the log file as well?

Reply:
Yes

Reply:
Then you want this change.